src/routes/user.route.js

```diff
@@ -2,11 +2,11 @@ import express from 'express';
 import passport from 'passport';
 import fileUpload from 'express-fileupload';
 import path from 'path';
-import UserController from '../controllers/users.controller';
+import UsersController from '../controllers/users.controller';
 import email from '../controllers/emails.controller';
 import checkRole from '../validation/checkRoles';
 import role from '../controllers/roles.controller';
```

Comment:
Change this to `RolesController`, and let this change reflect in other areas of the application.